js/background.js

```diff
@@ -133,54 +131,51 @@
 			domain: domain
 		});
 	}
-	
-	function updateLastAutoToast(domain, now) {
-		localforage.getItem(domain).then(function (item) {
-			if(item !== null){
-				item.lastAutoToastDate = now;
-				localforage.setItem(domain, item );
-			}
-		});
-	}
 
 	function storePrivacyScore(domain, data) {
 		localforage.setItem(domain, data);
 	}
 
-	function sendDataToContentScript(tabId, domain, data, now){
+	function sendDataToContentScriptActive(tabId, domain, data){
 		chrome.browserAction.setIcon({
 			path : {
 				'48': 'css/images/iconColored.png'
 			},
 			tabId : tabId
 		});
 
-		if(data.lastAutoToastDate === null || data.lastAutoToastDate === now || hasExpired(data.lastAutoToastDate, now)){
-			updateLastAutoToast(domain, now);
-			chrome.tabs.sendMessage( tabId, {
-				message: 'ActiveTabScore',
-				score: data.score,
-				previousScore: data.previousScore,
-				domain: domain
-			});
-		} else {
-			chrome.tabs.sendMessage( tabId, {
-				message: 'HiddenTabScore',
-				score: data.score,
-				previousScore: data.previousScore,
-				domain: domain
-			});
-		}
+		chrome.tabs.sendMessage( tabId, {
+			message: 'ActiveTabScore',
+			score: data.score,
+			previousScore: data.previousScore,
+			domain: domain
+		});
+	}
+
+	function sendDataToContentScriptHidden(tabId, domain, data){
+		chrome.browserAction.setIcon({
+			path : {
+				'48': 'css/images/iconColored.png'
+			},
+			tabId : tabId
+		});
+
+		chrome.tabs.sendMessage( tabId, {
+			message: 'HiddenTabScore',
+			score: data.score,
+			previousScore: data.previousScore,
+			domain: domain
+		});
 	}
 
-	function hasExpired(then, now) {
+	function hasExpired(then) {
 		const expireTime = Number(then) + (expiration.lastScoreExpirationDays * 86400 * 1000);
```

Comment:
@L0key this looks like it has the same calculation has the hasNoScoreExpired... Supposed to be 30 days?